cri/stream/portforward/httpstream.go

```diff
@@ -74,8 +74,8 @@ func handleHTTPStreams(ctx context.Context, w http.ResponseWriter, req *http.Req
 		streamChan:            streamChan,
 		streamPairs:           collect.NewSafeMap(),
 		streamCreationTimeout: streamCreationTimeout,
-		pod:       podName,
-		forwarder: portForwarder,
+		pod:                   podName,
```

Comment:
I am just curious about why the integrated CI has not detected this part? @houstar @fuweid @Starnop 😕

Reply:
I think it is bug in the gometerlinter. The megachecker process was killed by OOM, which might cause the following the check doesn't work. @allencloud